### PR TITLE
symbolic icon, right panel placement

### PR DESCRIPTION
**Pull request by itmitica**
Custom symbolic icon for force quit and placement to the right panel.
fqitmitica

### CODE REVIEW

File: extension.js [hunk 1]
```diff
@@ -23,11 +26,9 @@ function init() {
 }
 
 function enable() {
-	let appMenu=Main.panel.statusArea.appMenu.actor.get_parent();
```
Comment (meghprkh) [inline extension.js]: Comment our these 3 lines instead oof removing them please. For people who want to default to the old style (without implementing a preferences dialog)

**Comment by meghprkh** [comment]
Thanks

**Comment by itmitica** [comment]
Thank you for Force Quit!

One fix remains: `function disable()` _`leftBox` old style commented, I added the fix after you committed the pull request.

**Comment by meghprkh** [comment]
I fixed that and submitted extension for code review. Shall be updated on website in a few days. Tanks again

**Pull request merged**